src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -56,8 +58,20 @@ class EmojiPickerMenu extends Component {
 
         this.renderItem = this.renderItem.bind(this);
         this.isMobileLandscape = this.isMobileLandscape.bind(this);
+        this.updatePreferredSkinTone = this.updatePreferredSkinTone.bind(this);
+        this.onEmojiSelected = this.onEmojiSelected.bind(this);
     }
 
+    /**
+     * Callback for the emoji picker to add whatever emoji is chosen into the main input
+     *
+     * @param {String} emoji
+     * @param {Object} emojiObject
+     */
+    onEmojiSelected(emoji, emojiObject) {
```

Comment:
Method name should reflect what is happening in the function. `addToFrequentAndSelectEmoji`